README.md

```diff
@@ -39,6 +39,7 @@ Table of Contents:
 * [Appwrite + Flutter EP02: Wireframing](https://www.youtube.com/watch?v=RjE0tmyBdow)
 * [Appwrite + Flutter EP03: New Flutter project, UI designs](https://www.youtube.com/watch?v=HvcemJhSeE8)
 * [Appwrite + Flutter EP04: Let's Authenticate Users](https://www.youtube.com/watch?v=WcGQDmuwGMM)
+* [Appwrite + Flutter EP4.1: Quick fix for Permission Error](https://www.youtube.com/watch?v=EBevk7dfTjY&list=PLUiueC0kTFqI9WIeUSkKvM-a_3fyaIiuk)
```

Comment:
Remove the end of link `&list=PLUiueC0kTFqI9WIeUSkKvM-a_3fyaIiuk`

Reply:
Done, you can merge it now. :)